src/CodeGen.cpp

```diff
@@ -174,7 +174,7 @@ bool CodeGen::llvm_NVPTX_enabled = false;
 bool CodeGen::llvm_Mips_enabled = false;
 
 void CodeGen::compile(Stmt stmt, string name,
-                      const vector<Argument> &args,
+                      const ArgInfo &arg_info,
```

Comment:
It looks good to me as is, never mind the original comment. Just a suggestion in case you hadn't considered it.